### PR TITLE
Add Shared with me & Liked page

### PR DESCRIPTION
**Pull request by SaraVieira**
Adds two new pages on dashboard:

* Sandboxes shared with you:

CleanShot 2021-03-08 at 15 38 49

* Liked sandboxes:

CleanShot 2021-03-08 at 17 16 57

Waiting on backend deploy for PR Preview

### CODE REVIEW

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/5565/commits/94b517240f22e2dc87be04bd635e8e39a6284f86 failed.

File: packages/app/src/app/pages/Dashboard/Content/routes/Shared/index.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    : [{ type: 'skeleton-row' }, { type: 'skeleton-row' }];
+
+  const pageType: PageTypes = 'shared';
+  return (
+    <SelectionProvider page={pageType} activeTeamId={activeTeam} items={items}>
+      <Helmet>
+        <title>Shared with Me - CodeSandbox</title>
+      </Helmet>
+      <Header
+        title="Sanboxes Shared with Me"
```
Comment (CompuIves) [inline packages/app/src/app/pages/Dashboard/Content/routes/Shared/index.tsx:38]: Small typo!
```suggestion
        title="Sandboxes Shared with Me"
```

File: packages/app/src/app/pages/Dashboard/Content/routes/Liked/index.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      </Helmet>
+      <Header
+        title="Liked Sandboxes"
+        activeTeam={activeTeam}
+        templates={getPossibleTemplates(sandboxes.LIKED)}
+        showViewOptions
+        showFilters
+      />
+
+      <VariableGrid items={items} page={pageType} />
```
Comment (CompuIves) [inline packages/app/src/app/pages/Dashboard/Content/routes/Liked/index.tsx:45]: Do we define somewhere that these sandboxes are "read-only" eg. that we can't drag them or do actions on them? I haven't tested yet, not sure if we define it in code somewhere.

**Review by CompuIves — APPROVED** [review]
Code looks very good! I found a very small typo and I have one question, I haven't tested yet, but code looks good!

File: packages/app/src/app/pages/Dashboard/Content/routes/Liked/index.tsx (hunk 2 above)
Comment (SaraVieira) [reply packages/app/src/app/pages/Dashboard/Content/routes/Liked/index.tsx:45]: We check in the menu if the author is equal to the login user so it all happens automatically

Comment (CompuIves) [reply packages/app/src/app/pages/Dashboard/Content/routes/Liked/index.tsx:45]: ❤️ 🎉  Super nice!

**Comment by DannyRuchtie** [comment]
Would be super nice for the future to add a Like, unlike button in the context menu. 
Not sure how much work that is but noticed when using it I wanted to remove some sandboxes from my likes

**Pull request merged**